src/app/store/slices/storage/storage.thunks/uploadFolderThunk.ts

```diff
@@ -313,9 +323,10 @@ function countItemsUnderRoot(root: IRoot): number {
   while (queueOfFolders.length > 0) {
     const folder = queueOfFolders.shift() as IRoot;
 
-    count += folder.childrenFiles?.length ?? 0;
+    count += folder.childrenFiles.length;
 
     if (folder.childrenFolders) {
+      count += folder.childrenFolders.length;
```

Comment:
Folders are not uploaded to the network, won't this change the upload % when there are many subfolders to upload?

Reply:
This is what is required in the second part of the issue.